cmd/acr/login.go

```diff
@@ -62,23 +66,32 @@ func newLoginCmd(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVarP(&opts.username, "username", "u", "", "the registry username")
 	cmd.Flags().StringVarP(&opts.password, "password", "p", "", "the registry password or identity token")
 	cmd.Flags().BoolVarP(&opts.fromStdin, "password-stdin", "", false, "read password or identity token from stdin")
+	cmd.Flags().BoolVarP(&opts.insecure, "insecure", "k", false, "allow connections to SSL registry without certs")
+	cmd.Flags().BoolVarP(&opts.plainHTTP, "plain-http", "", false, "allow insecure connections to registry without SSL")
+	cmd.Flags().BoolVarP(&opts.verbose, "verbose", "v", false, "verbose output")
 	return cmd
 }
 
-func runLogin(opts loginOpts) error {
+func runLogin(opts loginOpts) (err error) {
```

Comment:
What's the purpose to have `(err error)`? Is it intended?

Reply:
No it's not intended. I changed it back.